doc/sphinx/source/project/extras.rst

```diff
@@ -20,9 +30,60 @@ Homomorphic Encryption (HE)
 
 
 
+.. _distributed_training:
+
+Distributed training with nGraph
```

Comment:
This section all looks good to me, but you might tag Wenzhe if you want a more expert opinion.

Reply:
Nothing new here, section is just moved to be categorized with the more fluid/ WIP pieces of nGraph.